pkgs/by-name/vp/vp/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  SDL,
+  SDL_image,
+  autoreconfHook,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vp";
+  version = "1.8-unstable-2017-03-22";
+
+  src = fetchFromGitHub {
+    owner = "erikg";
+    repo = "vp";
+    rev = "52bae15955dbd7270cc906af59bb0fe821a01f27";
+    hash = "sha256-AWRJ//0z97EwvQ00qWDjVeZrPrKnRMOXn4RagdVrcFc=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    SDL
+  ];
+
+  buildInputs = [
+    SDL
+    SDL_image
+  ];
+
+  outputs = [ "out" "man" ];
+
+  strictDeps = true;
+
+  env.NIX_CFLAGS_COMPILE = toString [
+    "-I${lib.getDev SDL}/include/SDL"
+    "-I${lib.getDev SDL_image}/include/SDL"
+  ];
+
+  meta = {
+    homepage = "https://github.com/erikg/vp";
+    description = "SDL based picture viewer/slideshow";
+    license  = lib.licenses.gpl3Plus;
+    mainProgram = "vp";
+    maintainers = with lib.maintainers; [ AndersonTores ];
```

Comment:
```
       error: undefined variable 'AndersonTores'

       at /home/pederbs/.cache/nixpkgs-review/pr-308379-7/nixpkgs/pkgs/by-name/vp/vp/package.nix:45:43:

           44|     mainProgram = "vp";
           45|     maintainers = with lib.maintainers; [ AndersonTores ];
             |                                           ^
           46|     inherit (SDL.meta) platforms;
```

Reply:
I think it's better to use the `a -> b` commit message schema, even if it is a downgrade

Reply:
This is not even a downgrade, since the version was misleading. But OK.